README.md

```diff
@@ -687,6 +687,7 @@ For APIs: There is a separate repository. Please visit [nepali-ukhaan-apis](http
 फिनी हराएको थाहा छैन गहुँको दाना गन्ने    |   	Fini harayeko thaha chhaina gahu ko dana ganne.
 फोकट को पाए एक मानो चुक पनी खान्छ    |   	Fokat ko paaye ek maano chuk pani khaanchha.
 फोश्रो आशा न देखाऊ नु | fosre asha na dekhaunu |  Promise only if you can make it.
+फुटेको खप्पर जहां गए पनि ठक्कर | futeko khappar jaha gaye pani thakkar | The broken pot gets bumped wherever it goes

```

Comment:
To add the Nepali proverb "फुटेको खप्पर जहां गए पनि ठक्कर" (phuteko khappar jaha gaye pani thakkar) to a list, ensure it's placed alphabetically between "फि" (phi) and "फो" (pho).  

The ordering is:
फ (pha)
फा (phaa)
फि (phi)
फी (phee)
फु (phu)
फू (phoo)
फे (phe)
फै (phai)
फो (pho)
फौ (phau)
फं (pham)
फः (phah)